### PR TITLE
CI runtime check cleanup

### PR DESCRIPTION
**Pull request by Ferroin**
##### Summary

- Centralize the logic used to perform basic runtime checks for builds in our CI.
- Expand the runtime checks to include verifying that the individual dashboards can be fetched.

##### Test Plan

CI passes on this PR.

### CODE REVIEW

File: packaging/runtime-check.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+wait_for() {
```
Comment (tkatsoulas) [inline packaging/runtime-check.sh:3]: maybe a combination of an exponential backoff or and retry logic?

Comment (tkatsoulas) [reply packaging/runtime-check.sh:3]: I know that more than 30 sec for a startup is bad, but with runners we can't be certain.

**Comment by tkatsoulas** [comment]
Other than that https://github.com/netdata/netdata/pull/16713#discussion_r1455380055 , looks good

File: packaging/runtime-check.sh (hunk 1 above)
Comment (Ferroin) [reply packaging/runtime-check.sh:3]: No, we can be even in this case. The way they run doesn’t result in long random delays, and 30 seconds is sufficient headroom for expected short delays. Keep in mind that this is a clean install with no data to load, so the startup time here is just however long it takes Netdata to get the internal data structures and the thread/process tree set up, which should realistically never be more than a single second on decent hardware.

Comment (Ferroin) [reply packaging/runtime-check.sh:3]: Also, to clarify a bit, a simple flat timeout was chosen here to intentionally make the script simpler (and thus less likely to have errors) and ensure that failures still register relatively quickly when they occur.

**Review by tkatsoulas — APPROVED** [review]

**Pull request merged**